### ISSUE TITLE
thinks about roles for the networks

### ISSUE THREAD

**Issue by albjeremias** [helpbuttons/hb-back#4]
- admin
 - moderator
 - user
 - blocked

**Comment by albjeremias** [comment]
@watchoutfreedom does this make sense?

**Comment by watchoutfreedom** [comment]
yes, it would be good to define this alongside the user model. Some features like "support" other users can affect those categories but I feel those 4 are the main important ones.

**Comment by watchoutfreedom** [comment]
When are we defining the user model?

**Comment by albjeremias** [comment]
> When are we defining the user model?

https://github.com/helpbuttons/hb-back/issues/9

**Comment by albjeremias** [comment]
this will define maybe api endpoints, and also is verrry related with the ACL. which means  #10 and #25 need to be implemented first

**Comment by albjeremias** [comment]
https://thomasvds.com/building-an-advanced-rbac-system-in-nest-js/
